src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs

```diff
@@ -54,12 +55,13 @@ public void Setup()
 
             // Mock successful token result
             this.tokenResult = new TokenResult(new JsonWebToken(TokenResultTest.FakeToken), Guid.NewGuid());
+            this.timeoutManager = new TimeoutManager(TimeSpan.FromSeconds(60));

```

Comment:
None of these tests take advantage of the `ITimeoutManager` abstraction. This is a real `TimeoutManager`, which means all of these tests are capped at 60s runtime (that's good anyway, but maybe not what you intend to express).

It's not clear to me what the `ITimeoutManager` abstraction is meant to help with. Maybe we don't need it?